clang/include/clang-c/Index.h

```diff
@@ -6953,7 +6953,6 @@ clang_getCursorUnaryOperatorKind(CXCursor cursor);
  * @}
  */
 
-CINDEX_DEPRECATED
```

Comment:
How about we go with `/* CINDEX_DEPRECATED */` and maybe a comment explaining that the type is deprecated but not marked so due to deprecation diagnostic behavior on MSVC?